internal/pkg/log/debug.go

```diff
@@ -57,6 +57,7 @@ func NewDebugLoggerWithPrefix(prefix string) DebugLogger {
 		cores = cores.With([]zapcore.Field{{Key: "prefix", String: prefix, Type: zapcore.StringType}})
 	}
 	l.zapLogger = loggerFromZapCore(cores)
+	l.zapLogger.prefix = prefix
```

Comment:
Nefungovalo mi `DebugLogger.AddPrefix`.

Ono je to trocha pomotane, je to implementovane tu:
https://github.com/keboola/keboola-as-code/blob/ef2ed14626344ba98d89f041efcabca38fffaf1a/internal/pkg/log/logger.go#L26-L32

A musi sa to uchovavt aj v tom fielde `prefix`, pretoze ked pridavas dalsi prefix, tak potrebujes vediet ten predchadzajuci, a ten sa neda vytiahnut z `zap` libky, ta ma iba setter.

`AddPrefix` funguje spravne, no chybalo nastavenie pociatocneho prefixu na tomto mieste.